data_service/src/match_processing/match_parser.rs

```diff
@@ -1,4 +1,4 @@
-use crate::types::{ChampionFrame, MatchTimelineFrame, PuuidToChampionMapping};
+use crate::types::{ChampionFrame, MatchTimelineFrame, ParticipantIdToChampionMapping};
 use chrono::{DateTime, TimeDelta};
 use riven::models::match_v5::{
     Match, MatchTimeline, MatchTimelineInfoFrameParticipantFrame, Participant,
```

Comment:
now parse_match_timeline should accept match_timeline and a participant_to_champion mapping